## 03_allow_spaces_in_zip_filenames.dpatch by Florian Ernst <florian@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Allow spaces in filenames for zipped files, bug#238678

@DPATCH@
diff -urNad uae-0.8.25~/src/zfile.c uae-0.8.25/src/zfile.c
--- uae-0.8.25~/src/zfile.c	2005-07-01 17:53:26.000000000 +0200
+++ uae-0.8.25/src/zfile.c	2005-11-08 01:17:59.000000000 +0100
@@ -126,7 +126,7 @@
     if (!dst)
 	return 1;
 #if defined AMIGA || defined __unix
-    sprintf (cmd, "unzip -p %s '*.adf' >%s", src, dst);
+    sprintf (cmd, "unzip -p \"%s\" '*.adf' >%s", src, dst);
     return !system (cmd);
 #endif
 }
